openresty-patches/patches/1.15.8.3/LuaJIT-2.1-20190507_03-dyn-lightuserdata-map.patch

```diff
@@ -182,7 +183,9 @@ index a0fba2a..1ab3097 100644
 +
 +  free(p);
 +
-+  luvmap_fold();
++  if (!luvmap_fold()) {
```

Comment:
This (plus the replacement of `abort()` by `return NULL`) feels like an extra change, in addition to the change in size. Depending on how different this is from the previous behavior (which I have no way to tell) we might want to include a note in the PR about it, or even add a separate commit for it.

Reply:
the behaviour is the same: refuse to start if it can't find a suitable mapping.  the difference is that doing `abort()` results in a coredump, while now it just returns a non-success result.

I'd like to put a nice error message, but that implies either pulling `<stdio.h>` on this file (which currently only does internal-structs handling), or a much more invasive patch to add new failure modes.